benchmark/timers/immediate.js

```diff
@@ -2,113 +2,113 @@
 const common = require('../common.js');
 
 const bench = common.createBenchmark(main, {
-  thousands: [5000],
+  n: [5000e3],
   type: ['depth', 'depth1', 'breadth', 'breadth1', 'breadth4', 'clear']
 });
 
-function main({ thousands, type }) {
-  const N = thousands * 1e3;
+function main({ n, type }) {
+
   switch (type) {
     case 'depth':
-      depth(N);
+      depth(n);
       break;
     case 'depth1':
-      depth1(N);
+      depth1(n);
       break;
     case 'breadth':
-      breadth(N);
+      breadth(n);
       break;
     case 'breadth1':
-      breadth1(N);
+      breadth1(n);
       break;
     case 'breadth4':
-      breadth4(N);
+      breadth4(n);
       break;
     case 'clear':
-      clear(N);
+      clear(n);
       break;
   }
 }
 
 // setImmediate tail recursion, 0 arguments
-function depth(N) {
-  var n = 0;
+function depth(n) {
+  var i = 0;
```

Comment:
This is a independent change and is not necessary. Please make it as it was before. This applies to all these functions in this test.